src/Host/HostSettings.cs

```diff
@@ -17,7 +17,7 @@ public class HostSettings
 
         // DBs
         public string UsePostgreSql { get; set; } =
-            "Server=localhost;Database=board_games_dev;Port=5432;User Id=postgres;Password=Fusion.0.to.1";
+            "Server=localhost;Database=board_games_dev;Port=5432;User Id=postgres;Password=pg051825";
```

Comment:
IMO you shouldn't change the default password that matches the password from docker-compose.yml

Reply:
Maybe just use the original docker-compose.yml?